### ISSUE TITLE
Arm64 Support

### ISSUE THREAD

**Issue by argentini** [xoofx/SharpScss#19]
Are there plans for Arm64 support (e.g. Apple Silicon)?

**Comment by xoofx** [comment]
Unlikely before [GitHub actions support it for their virtual environments](https://github.com/actions/virtual-environments/issues/2187).

**Comment by saxx** [comment]
Is there any way to get SharpScss running on an M1 Mac (with .NET 6 for ARM)? It seems I cannot get past this error: 
````
System.DllNotFoundException: Unable to load shared library 'libsass' or one of its dependencies. In order to help diagnose loading problems, consider setting the DYLD_PRINT_LIBRARIES environment variable: dlopen(liblibsass, 0x0001): tried: 'liblibsass' (no such file), '/usr/local/lib/liblibsass' (no such file), '/usr/lib/liblibsass' (no such file), '/Users/hannes/Projects/Igevia/src/backend/liblibsass' (no such file), '/usr/local/lib/liblibsass' (no such file), '/usr/lib/liblibsass' (no such file)
````

**Comment by saxx** [comment]
Follow-Up on my last post: I got it running on an M1 Mac (.NET 6 for ARM) by downloading the latest `libsass` version via homebrew, then I dropped the `libsass.dylib` from the homebrew directory into the `bin` directory of my application. 

SharpScss works now :) This is in no way ideal, but at least it's running for now. Thank you for this great library.

**Comment by danroot** [comment]
Ran into this just now and was stumped.  Thanks to @saxx I was able to do this to resolve for now:

```
brew install libsass
cp /opt/homebrew/Cellar/libsass/3.6.5/lib/libsass.dylib ./bin/Debug/net6.0/libsass.dylib
```
I'll probably add to my .net code to do this automatically if in an ARM environment.

I second a switch to dartsass or some other cross-platform sass engine that can "just work" Mac M1/M2 (or conceivably other ARM environments).

**Comment by xoofx** [comment]
Fixed in `2.1.0`+
